Byttet funksjonene expect_*(f(), NA) til expect_no_*(f))

orjanls opened the pull request.

orjanls committed:
Byttet funksjonene expect_*(f(), NA) til expect_no_*(f))

haugedal reviewed (changes requested):
Ser bra ut 👍

I linje 39 i `test-anonymiser.R`, som me såg litt på tidlegare, var truleg `NA` plassert feil i utgangspunktet. `NA` skulle nok vore i `expect_error()` og ikkje `suppressWarnings()`. Me kan gjerne berre retta det med ein gong, altså byta til `expect_no_error()` og fjerna `NA`.

Commit:
Rettet feil-implementering av test

NA var plassert i `suppressWarnings()`, når den skulle vært i
`expect_error()`. Rettet dette og forandret funksjonen fra
`expect_error(f(), NA)` til `expect_no_error(f()).

orjanls commented:
Nå er linje 39 i  `test-anonymiser.R` oppdatert til expect_no_error().

haugedal commented:
👍

Loggmeldinga i siste commit bør forklara at testen var implementert feil før (feilplassert `NA`). Bruk det ekstra avsnittet i loggmeldinga for ekstra detaljar når du treng meir enn éi linje.

Du kan enkelt oppdatera den siste loggmeldinga ved å køyra `git commit --amend` (og så `git push --force-with-lease`).
